modelzoo/rnnt/pytorch/rnnt.patch

```diff
@@ -4749,14 +4767,28 @@ index de45eb0..8bb5542 100644
 -            print_once(f'Finished after {args.epochs_this_job} epochs.')
 -            break
 -        # end of epoch
- 
+-
++                    audio, audio_lens, txt, txt_lens = batch
++                    feats, feat_lens = train_feat_proc([audio, audio_lens])
++                    meta_data = []
++                    B_split = batch_size // args.batch_split_factor
++                    for i in range(args.batch_split_factor):
++                        meta_data.append(train_preproc.get_packing_meta_data(   feats.size(0),
++                                                                                feat_lens[i*B_split:(i+1)*B_split],
++                                                                                txt_lens[i*B_split:(i+1)*B_split]))
++
++                if args.enable_seq_len_stats:
++                    all_feat_lens += feat_lens
++
++
++
 +                loss_item= train_step( model, loss_fn, args, batch_size, feats, feat_lens, txt, txt_lens,
 +                                        meta_data, train_loader, rnnt_graph)
 +                step_utts += txt_lens.size(0) * world_size
 +                epoch_utts += txt_lens.size(0) * world_size
 +                accumulated_batches += 1
 +                if accumulated_batches % args.grad_accumulation_steps == 0:
-+
++                    torch.nn.utils.clip_grad_norm_(model.parameters(), args.clip_norm)
```

Comment:
args.clip_norm is passed through config or cmdline?

Reply:
`args.clip_norm` is a cmdline arg passed in `train.sh`